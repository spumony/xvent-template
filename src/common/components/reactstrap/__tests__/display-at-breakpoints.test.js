import React from 'react';
import { render } from '@testing-library/react';

import DisplayAtBreakpoints from '../display-at-breakpoints';

describe('display-at-breakpoints', () => {
  it('should render properly', () => {
    const { container } = render(
      <DisplayAtBreakpoints breakpoints={['md', 'lg']}>
        <p>This text is visible only at md and lg breakpoint</p>
      </DisplayAtBreakpoints>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
