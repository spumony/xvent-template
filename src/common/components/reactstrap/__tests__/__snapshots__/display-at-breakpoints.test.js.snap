// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display-at-breakpoints should render properly 1`] = `
<div
  class="d-none d-md-block d-lg-block"
>
  <p>
    This text is visible only at md and lg breakpoint
  </p>
</div>
`;
